### PR TITLE
Feature/lantern 116

### PR DESCRIPTION
**Pull request by bmath10**
Testing:
-  While `docker-compose up` is running, in `endpointmanager/` run: `go run cmd/endptquerier/main.go ../endpoints/resources/EndpointSources.json`
- Run `docker exec -it pg_prometheus psql -U lantern` to access the database
- Running `select count(*) from fhir_endpoints;` should return 341

**I ended up adding OrganizationName but not Location to the fhir_endpoints database, so technically [LANTERN-118](https://oncprojectracking.healthit.gov/support/browse/LANTERN-118) has been started**

--------------------

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-116
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:** @mayerm94 

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

### CODE REVIEW

File: endpointmanager/cmd/endptquerier/main.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		log.Fatalf("%s %s", errString, err)
+	}
+}
+
+func main() {
+	var endpointsFile string
+	if len(os.Args) != 1 {
+		endpointsFile = os.Args[1]
+	} else {
+		failOnError("ERROR: Missing endpoints list command-line argument", nil)
```
Comment (edeyoung) [inline endpointmanager/cmd/endptquerier/main.go]: i don't think this error will occur as expected because the `err` argument is nil

File: endpointmanager/cmd/endptquerier/main.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	err = config.SetupConfig()
+	failOnError("", err)
+
+	ctx := context.Background()
+	store, err := postgresql.NewStore(viper.GetString("dbhost"), viper.GetInt("dbport"), viper.GetString("dbuser"), viper.GetString("dbpassword"), viper.GetString("dbname"), viper.GetString("dbsslmode"))
+	failOnError("", err)
+	log.Info("Successfully connected to DB!")
+
+	dbErr := endptQuerier.AddEndpointData(ctx, store, &listOfEndpoints)
+	failOnError("Saving in fhir_endpoints database error: ", dbErr)
```
Comment (edeyoung) [inline endpointmanager/cmd/endptquerier/main.go]: we need to standardize our go error handling. i'll add to the sprint review agenda. no request for changes - just an observation.

Comment (bmath10) [reply endpointmanager/cmd/endptquerier/main.go]: 👍

File: endpointmanager/cmd/endptquerier/main.go (hunk 1 above)
Comment (bmath10) [reply endpointmanager/cmd/endptquerier/main.go]: This has now been updated to make it clearer.

File: endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	if err == sql.ErrNoRows {
+		err = store.AddFHIREndpoint(ctx, fhirEndpoint)
+		if err != nil {
+			return errors.Wrap(err, "adding fhir endpoint to store failed")
+		}
+	} else if err != nil {
+		return errors.Wrap(err, "getting fhir endpoint from store failed")
+	} else {
+		// Currently just logging if there is a repeat
+		log.Info(existingEndpt)
```
Comment (edeyoung) [inline endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints.go]: make log message more explanatory: "An endpoint for <...> has already been added to the database."

Comment (bmath10) [reply endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints.go]: 👍
